exercises/option/option1.rs

```diff
@@ -0,0 +1,23 @@
+//option1.rs
+//Make me compile! Execute `rustlings hint option1` for hints
+
+//I AM NOT DONE
+
+//you can modify anything EXCEPT for this function's sig
```

Comment:
`sig -> signature`, I wouldn't assume people know what that means

Reply:
Hmm I guess so, how should I signify that people shouldn't modify the function header?